media_kit_video/common/darwin/Classes/plugin/VideoOutput.swift

```diff
@@ -185,31 +177,23 @@ public class VideoOutput: NSObject {
     }
   }
 
-  private var videoWidth: Int64 {
-    // fixed width
-    if width != nil {
-      return width!
-    }
-
-    let params = MPVHelpers.getVideoOutParams(handle)
-    let width = params.rotate == 0 || params.rotate == 180
-      ? params.dw
-      : params.dh
-
-    return width
-  }
-
-  private var videoHeight: Int64 {
-    // fixed height
-    if height != nil {
-      return height!
-    }
-
-    let params = MPVHelpers.getVideoOutParams(handle)
-    let height = params.rotate == 0 || params.rotate == 180
-      ? params.dh
-      : params.dw
-
-    return height
+    private var videoSize: CGSize {
+        // fixed size
+        if width != nil && height != nil {
+            return CGSize(
+                width: Double(width!),
+                height: Double(height!)
+            )
+        }
+        
+        let params = MPVHelpers.getVideoOutParams(handle)
+        return CGSize(
+            width: Double(width ?? (params.rotate == 0 || params.rotate == 180
+                                    ? params.dh
+                                    : params.dw)),
+            height: Double(height ?? (params.rotate == 0 || params.rotate == 180
```

Comment:
@moffatman seems like a mistake?
Previously: width = params.rotate == 0 || params.rotate == 180 ? dw : dh
Now it's: width = params.rotate == 0 || params.rotate == 180 ? dh : dw (swapped)
Same for height.

Reply:
whoops https://github.com/media-kit/media-kit/pull/814